lbz/resource.py

```diff
@@ -117,7 +117,8 @@ def get_guest_authorization() -> dict:
         return {}
 
     def get_authz_data(self) -> dict:
-        return self._authz_collector.dump()
+        dump: dict = self._authz_collector.dump()
```

Comment:
That change doesn't make much sense because `.dump()` is our internal method and contains a type annotation - the type will be discovered automatically. I don't see any other reason to create a separate variable for that but if you see, please share it with me 😉